[2022-01-29 14:08:07,483] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: HomeWorkWeek2.get_data scheduled__2021-05-02T06:00:00+00:00 [queued]>
[2022-01-29 14:08:07,515] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: HomeWorkWeek2.get_data scheduled__2021-05-02T06:00:00+00:00 [queued]>
[2022-01-29 14:08:07,515] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 14:08:07,515] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-29 14:08:07,515] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 14:08:07,551] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): get_data> on 2021-05-02 06:00:00+00:00
[2022-01-29 14:08:07,556] {standard_task_runner.py:52} INFO - Started process 412 to run task
[2022-01-29 14:08:07,561] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'HomeWorkWeek2', 'get_data', 'scheduled__2021-05-02T06:00:00+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions1.py', '--cfg-path', '/tmp/tmp_jy1mf64', '--error-file', '/tmp/tmp6ty4rl5m']
[2022-01-29 14:08:07,561] {standard_task_runner.py:77} INFO - Job 46: Subtask get_data
[2022-01-29 14:08:07,868] {logging_mixin.py:109} INFO - Running <TaskInstance: HomeWorkWeek2.get_data scheduled__2021-05-02T06:00:00+00:00 [running]> on host d1e1fe005577
[2022-01-29 14:08:08,167] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=HomeWorkWeek2
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2021-05-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-02T06:00:00+00:00
[2022-01-29 14:08:08,168] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-29 14:08:08,169] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "helloworld!!"']
[2022-01-29 14:08:08,180] {subprocess.py:85} INFO - Output:
[2022-01-29 14:08:08,182] {subprocess.py:89} INFO - helloworld!!
[2022-01-29 14:08:08,183] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-29 14:08:08,246] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=HomeWorkWeek2, task_id=get_data, execution_date=20210502T060000, start_date=20220129T140807, end_date=20220129T140808
[2022-01-29 14:08:08,289] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-29 14:08:08,352] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-01-30 10:54:00,712] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: HomeWorkWeek2.get_data scheduled__2021-05-02T06:00:00+00:00 [queued]>
[2022-01-30 10:54:00,786] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: HomeWorkWeek2.get_data scheduled__2021-05-02T06:00:00+00:00 [queued]>
[2022-01-30 10:54:00,788] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-30 10:54:00,789] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-30 10:54:00,789] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-30 10:54:00,868] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): get_data> on 2021-05-02 06:00:00+00:00
[2022-01-30 10:54:00,878] {standard_task_runner.py:52} INFO - Started process 1111 to run task
[2022-01-30 10:54:00,900] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'HomeWorkWeek2', 'get_data', 'scheduled__2021-05-02T06:00:00+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions1.py', '--cfg-path', '/tmp/tmpeq7a59z5', '--error-file', '/tmp/tmp8v091rnc']
[2022-01-30 10:54:00,901] {standard_task_runner.py:77} INFO - Job 91: Subtask get_data
[2022-01-30 10:54:01,199] {logging_mixin.py:109} INFO - Running <TaskInstance: HomeWorkWeek2.get_data scheduled__2021-05-02T06:00:00+00:00 [running]> on host fc414ca9b482
[2022-01-30 10:54:01,431] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-01-30 10:54:01,539] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=HomeWorkWeek2
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2021-05-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-02T06:00:00+00:00
[2022-01-30 10:54:01,542] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-30 10:54:01,543] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSLf https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2021-05.csv > /opt/***output_2021-05.csv']
[2022-01-30 10:54:01,576] {subprocess.py:85} INFO - Output:
[2022-01-30 10:54:39,173] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-30 10:54:39,297] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=HomeWorkWeek2, task_id=get_data, execution_date=20210502T060000, start_date=20220130T105400, end_date=20220130T105439
[2022-01-30 10:54:39,415] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-30 10:54:39,525] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
